### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Lawmsangi**
On this project, I implemented the following: 

- used [Gutendex API](https://gutendex.com/)
- there are no linters error.
- used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- used react documentaion, react-redux, redux toolkit
-  wrote integration tests with a React testing library.
- used styles in a React app according to given design.
- used store, actions and reducers in React.
- added filter feature for book title.
- added details section for book title.
- added back button to navigate to the home page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,153 @@
...
+    
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://books-library-oy1a.onrender.com/)
+
+ 
+## 🎥 Live Video <a name="live-video"></a>
```
Comment (FrederickMih) [inline README.md:57]: - [ ] Hi @Lawmsangi, great job for taking your time to highlights the features of your project in the presentation. Once again, in the requirements of this project, we were suggested to equally highlight a piece of code we find interesting during the course of development. Kindly open your code editor during presentation and highlight the piece of code you're proud of making :+1:

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          width="1em"
+          xmlns="http://www.w3.org/2000/svg"
+          onClick={() => navigate('/')}
+        >
+          <path d="M217.9 256L345 129c9.4-9.4 9.4-24.6 0-33.9-9.4-9.4-24.6-9.3-34 0L167 239c-9.1 9.1-9.3 23.7-.7 33.1L310.9 417c4.7 4.7 10.9 7 17 7s12.3-2.3 17-7c9.4-9.4 9.4-24.6 0-33.9L217.9 256z" />
+        </svg>
+        <input
+          type="text"
+          onChange={handleInputChange}
+        />
```
Comment (FrederickMih) [inline src/components/Header.js]: - [Optional] For a great user's experience, I think we should be added a placeholder text :+1:

File: src/tests/App.test.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  },
+) => state;
+
+const store = configureStore({ reducer });
+describe('Render Test', () => {
+  it('Component renders correctly', () => {
+    const tree = render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <App />
```
Comment (FrederickMih) [inline src/tests/App.test.js:23]: - [ ] Based on the requirements of this project, let's kindly add integration tests using React Testing Library. You may perform a mocking data fetching and Redux Store connection :+1:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @Lawmsangi, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Highlights :green_circle: 

- App is responsive and design aligns with the given design :+1: 

## suggestions Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions
**N/A**
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by cynthiainga — APPROVED** [review]
Hi @Lawmsangi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] All requested changes are made ✔️ 
- [ ] No linters errors ✔️ 
- [ ] The app works as expected ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**